logstash-core/src/main/java/org/logstash/execution/AbstractPipelineExt.java

```diff
@@ -122,6 +122,9 @@ public class AbstractPipelineExt extends RubyBasicObject {
     private static final RubySymbol DROPPED_EVENTS =
             RubyUtil.RUBY.newSymbol("dropped_events");
 
+    private static final RubySymbol DISCARDED_EVENTS =
+            RubyUtil.RUBY.newSymbol("discarded_events");
```

Comment:
What do you think about calling this `expired_events`? "discarded" and "dropped" I think may be too similar, and I think "expired" may explain the reasoning better.

Reply:
I agree, thanks for hint